### PR TITLE
Tests/email notifications

### PR DESCRIPTION
**Pull request by marek-siemieniuk-morawski**
## What does this PR do?
- Add options to manipulate Gmail API for email subscription testing.
- Adds UI tests for subscribing email in: 
-- settings section
-- during onboarding
-- during creating a new wallet

#### *Any scout cleaning/refactoring?*
- File `helpers.ts` was separated to `const`, `types` and `utils`.
- File `addWallet.spec.ts` was separated to `general`, `withoutSubscribingToNotifications.spec` and `withSubscribingToNotifications.spec`.

#### *Any blog post related to the solution you have used?*
Details about Gmail API: https://developers.google.com/gmail/api/quickstart/nodejs

## Todos:
- completing tests with comments
- [x] Checked on iOS
- [ ] Checked on Android

### CODE REVIEW

**Review by aleparty — COMMENTED** [review]
lgtm but @KedziaPawel or other developer should take a look as well before merge

File: src/screens/ChamberOfSecrets.tsx
Comment (Kamill90) [inline src/screens/ChamberOfSecrets.tsx:72]: `${email.substr(0, atIndex)}+${randomNumberString}${email.substr(atIndex)}` ?

**Review by Kamill90 — APPROVED** [review]

File: scripts/createGmailApiToken.js
Comment (DawidBastek) [reply scripts/createGmailApiToken.js:2]: I will leave it as is.

**Pull request merged**
